src/modules/APIs.js

```diff
@@ -0,0 +1,54 @@
+// I run this code in console when I want to reset the board by creating a id for involvement api
+
+// fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+//   method: 'POST',
+//   body: JSON.stringify({
+//     name: 'comment',
+//   }),
+//   headers: {
+//     'Content-type': 'application/json; charset=UTF-8',
+//   },
+// })
+//   .then((response) => response.json())
+//   .then((json) => console.log(json));
+
+// the id I get running the previous fetch code
+// const ID = 'woTq8RsLkSavLaCMQAAP';
+
+// APIs list
+const pokemonAPI = 'https://pokeapi.co/api/v2/pokemon/';
+const commentsAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/woTq8RsLkSavLaCMQAAP/comments/';
+const likesAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/woTq8RsLkSavLaCMQAAP/likes/';
+
+// onst pokemonList = document.getElementById('pokemonList');
+
+// this will get an api array
+
+const GetDataList = async (APIUrl) => {
+  const response = await fetch(APIUrl);
+  const dataList = await response.json();
+  const itemsArray = dataList.results;
+  return itemsArray;
+};
+
+const GetDataObject = async (APIUrl) => {
+  const response = await fetch(APIUrl);
+  const dataList = await response.json();
+  return dataList;
+};
+
+// to get the pokemon Data
+GetDataList(pokemonAPI);
+
```

Comment:
- [ ] Please refactor your fetching function from the external API because it seems it's not working. See the image down below. 🙏 
![-htmlWebpackPlugin-options-title-](https://github.com/GeekyHacks/Module2Capstone_Pokemon/assets/47625407/3ac4f8c4-1bb2-41ee-bb92-2af158a89f68)